<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Research Skills &amp; Methodologies</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>751</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Source</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>40</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose the number of tasks in one set:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Load file</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInfo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>551</width>
        <height>271</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>pwr.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>320</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Author:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>330</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Jaros≈Çaw Szumega</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>290</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Research skills and methodologies</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>280</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Course:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Or load your own dataset:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>My own data</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Simulated Annealing</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Cooling method</string>
      </property>
      <widget class="QRadioButton" name="radioLinear">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Linear</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioLogar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Logarithmic</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="linBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001000000000000</double>
       </property>
       <property name="maximum">
        <double>0.999000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>0.999000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>191</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Annealing parameters</string>
      </property>
      <widget class="QDoubleSpinBox" name="minTbox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="maxTbox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="value">
        <double>40.000000000000000</double>
       </property>
      </widget>
      <widget class="QSpinBox" name="stepBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelMin">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min Temperature</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelmax">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max Temperature</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelStp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Inner iterations</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="solveButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Solve!</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>441</width>
        <height>311</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Calculation range</string>
      </property>
      <widget class="QSpinBox" name="fromBox1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="ToBox1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelMin_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>From</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMin_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Show solution</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Invasion Weed</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>251</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Invasion parameters</string>
      </property>
      <widget class="QLabel" name="labelGen">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>No. generations</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="genBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="value">
        <double>60.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="maxWeedBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>25.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="labelGen_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>No. maximum weeds</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="minSeedBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>140</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="labelGen_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min.  seeds per weed</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="mutBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>220</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.800000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="labelGen_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Mutation probability</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelGen_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max. seeds per weed</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="maxSeedBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>20.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="popBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="value">
        <double>20.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="labelGen_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Initial population</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="solveButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Solve!</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>441</width>
        <height>311</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Calculation range</string>
      </property>
      <widget class="QSpinBox" name="FromBox2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="ToBox2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelMin_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>From</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMin_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Show solution</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
